roles/ceph-rgw-loadbalancer/defaults/main.yml

```diff
@@ -12,6 +12,10 @@ haproxy_frontend_port: 80
 haproxy_frontend_ssl_port: 443
 haproxy_frontend_ssl_termination: False
 
+haproxy_stats_socket_enabled: false
```

Comment:
I'd vote to just exclude this entirely - the stats socket exists already, just had different owner/permissions then we want. No need to add a flag for it.

Reply:
Done.